app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function `copy_file` should handle cases where the command is not in the correct format. Consider adding validation to ensure the command starts with 'cp' and has exactly three parts.